app/views/admin/link_check_reports/_link_check_report.html.erb

```diff
@@ -19,7 +19,12 @@
       <p>We’ve found links in this document that may be broken:</p>
       <ul>
         <% report.broken_links.each do |link| %>
-          <li><%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %></li>
+          <li>
+            <%= link_to link.uri.truncate(50), link.uri, title: link.uri, class: 'link-inherit' %>
+            <% if link.suggested_fix %>
+              <p class="suggested-fix"><%= link.problem_summary %>. Suggested fix: <%= link.suggested_fix %></p>
```

Comment:
I think it would be good to always show the problem summary, but only show the suggested fix if there is one.